tools/report.sh

```diff
@@ -81,6 +81,25 @@ if [ -z $namespace ]; then
    usage
 fi
 
+available_namespaces=`$platform get ns -o name`
+prefixed_ns="namespace/${namespace}"
+ns_avalailable=`printf '%s\n' "${available_namespaces[@]}"  | grep -P "^$prefixed_ns$"`
+
+if [ -z "${ns_avalailable}" ]; then
+	echo "Namespace $namespace not found! Exiting"
+	exit 1
+fi
```

Comment:
I think you can just do `kubectl get namespace <namespace>` and check the return code.

Reply:
Yeah. I am sure that is way as well. Using it as it does the code cleaner.